plugin.cmake

```diff
@@ -8,7 +8,10 @@ add_python_style_test(python_static_analysis_covalic
                       "${PROJECT_SOURCE_DIR}/plugins/covalic/server")
 
 add_eslint_test(
-  covalic "${PROJECT_SOURCE_DIR}/plugins/covalic/web_external/js"
-  ESLINT_CONFIG_FILE "${PROJECT_SOURCE_DIR}/plugins/covalic/web_external/js/.eslintrc"
-  ESLINT_IGNORE_FILE "${PROJECT_SOURCE_DIR}/plugins/covalic/web_external/js/.eslintignore"
+  covalic "${PROJECT_SOURCE_DIR}/plugins/covalic/web_external"
+  ESLINT_CONFIG_FILE "${PROJECT_SOURCE_DIR}/plugins/covalic/web_external/.eslintrc"
+  ESLINT_IGNORE_FILE "${PROJECT_SOURCE_DIR}/plugins/covalic/web_external/.eslintignore"
 )
+
+add_puglint_test(
+    covalic "${PROJECT_SOURCE_DIR}/plugins/${PLUGIN}/web_external/templates")
```

Comment:
`PLUGIN` is not defined in this file, but it could reduce duplication if we define it!